dtfs-central-api/src/helpers/fee-record-matching.test.ts

```diff
@@ -0,0 +1,119 @@
+import { Currency, FeeRecordEntityMockBuilder, PaymentEntityMockBuilder, UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
+import { feeRecordsAndPaymentsMatch } from './fee-record-matching';
+
+describe('fee-record-matching', () => {
+  describe('feeRecordsAndPaymentsMatch', () => {
+    it('returns true when the payments attached to the fee records have the same total payments', () => {
```

Comment:
I think this test name is slightly lacking (only referencing total payments not total fees paid to ukef) but then when I typed out a more in depth one it also didn't read well so I've resigned myself to leave this comment and just say this is fine